README.md

```diff
@@ -1 +1,37 @@
-LearnAway is a tool to facilitate the self-learning created by two technology enthusiasts, who live far from each other but linked by the passion for programming.
\ No newline at end of file
+# learnaway
+
+LearnAway is a tool to facilitate the self-learning created by two technology enthusiasts, who live far from each other but linked by the passion for programming.
+
+## learnaway-api
+
+### Requirements
+
+Requires docker-compose 
+
+```
+apt-get install docker-compose
+```
+
+### Deploy
+
+```
+docker-compose build
+docker-compose up -d
+docker-compose exec php bash
+```
+Inside php container in root workdir:
+
+```
+php bin/console server:run *:8080
```

Comment:
If you add this command as `ENTRYPOINT` in the `Dockerfile`, it will be executed on `docker run` (so that you don't need to do `exec php bash` + `php bin/console server:run *:8080`.